src/distribution/logistic.rs

```diff
@@ -0,0 +1,206 @@
+use distribution;
+use source::Source;
+
+/// A logistic distribution.
+#[derive(Clone, Copy)]
+pub struct Logistic {
+    mu: f64,
+    s: f64,
+}
+
+impl Logistic {
+    /// Create a logistic distribution with location `mu` and scale `s`.
+    ///
+    /// It should hold `s > 0`.
+    #[inline]
+    pub fn new(mu: f64, s: f64) -> Self {
+        should!(s > 0.0);
+        Logistic { mu: mu, s: s }
+    }
+
+    /// Return the location parameter.
+    #[inline(always)]
+    pub fn mu(&self) -> f64 { self.mu }
+
+    /// Return the scale parameter.
+    #[inline(always)]
+    pub fn s(&self) -> f64 { self.s }
+}
+
+impl Default for Logistic {
+    #[inline]
+    fn default() -> Self {
+        Logistic::new(0.0, 1.0)
```

Comment:
I’m not that familiar with the logistic distribution. Is there such thing as the standard logistic distribution (analogous to the standard Gaussian distribution, for instance)?